shared/profile/render.js.flow

```diff
@@ -19,6 +19,7 @@ export type Props = {
   onFollow: () => void,
   onUnfollow: () => void,
   onAcceptProofs: () => void,
+  onMissingProofClick: (proof: MissingProof) => void,
```

Comment:
:sheep: Might be helpful to name this `missingProof` so it's not confused with the different `Proof` type.